app/src/main/java/com/example/soundcloud/home/HomeFragmentPagerAdapter.java

```diff
@@ -8,22 +8,41 @@
 
 import com.example.soundcloud.data.TabInfo;
 import com.example.soundcloud.discover.DiscoverFragment;
+import com.example.soundcloud.favorite.FavoriteFragment;
+import com.example.soundcloud.my_music.MyMusicFragment;
 
 public class HomeFragmentPagerAdapter extends FragmentPagerAdapter {
     private static final int PAGE_COUNT = 3;
     private TabInfo[] mTabInfos;
     private Context mContext;
+    private Fragment mDiscoverFragment;
+    private Fragment mMyMusicFragment;
+    private Fragment mFavoriteFragment;
 
     public HomeFragmentPagerAdapter(Context context,
                                     FragmentManager fm,
                                     TabInfo[] tabInfos) {
         super(fm);
         mContext = context;
         mTabInfos = tabInfos;
+        mDiscoverFragment = new DiscoverFragment();
+        mMyMusicFragment = new MyMusicFragment();
+        mFavoriteFragment = new FavoriteFragment();
     }
 
     @Override
     public Fragment getItem(int i) {
+        switch (i){
+            case 0:
+                return mMyMusicFragment;
+
```

Comment:
38 41 44 add break
and add default case

Reply:
Nếu thêm break thì lệnh break sẽ unreachable vì nó ở bên duới return anh ạ.